### ISSUE TITLE
Add photos from your digital camera section enhancements

### ISSUE THREAD

**Issue by Tintels** [isaachinman/wundershine-web-app#17]
**DESKTOP**

Screenshot Webflow large display
![image](https://user-images.githubusercontent.com/22452702/48479109-990cc980-e806-11e8-8ed4-71019294ce36.png)

Screenshot live site large display
![image](https://user-images.githubusercontent.com/22452702/48479119-a1fd9b00-e806-11e8-997d-1ffc68dad075.png)

- [ ]  Move text block a bit to the left and lower. "Add" should be over dark trees.
- [x]  Text and icons allign middle
- [x]  Text block 500 px wide
- [x]  Resize icons set. Webflow: Icon set is 4 colums. Together 60% width of text block.
- [ ]  Icons width left to right: 50, 51, 46, 47 px.

**TABLET**

Screenshot target Webflow site on iPad Mini
![2018-11-14 12 13 19](https://user-images.githubusercontent.com/22452702/48479154-c3f71d80-e806-11e8-8e90-c41f5078866f.png)

Screenshot live site on iPad Mini
![2018-11-14 12 13 33](https://user-images.githubusercontent.com/22452702/48479156-c8233b00-e806-11e8-876a-19d0abfc0829.png)

- [x]  Move text block to the left and lower. "Add" should be over tall mountain peak.
- [x]  Text and icons allign middle
- [x]  Text block 400 px wide
- [x] Icons set style same as desktop

**PHONE**

Screenshot target Webflow site on Nexus 5X
![screenshot_20181114-121512](https://user-images.githubusercontent.com/22452702/48479233-07518c00-e807-11e8-8e2e-d67c62455f0e.png)

Screenshot live site on Nexus 5X
![screenshot_20181114-121527](https://user-images.githubusercontent.com/22452702/48479239-0ae51300-e807-11e8-8c37-54eb29221280.png)

- [x]  No vertical image crop
- [x]  Photographer should be fully  visible. Size and position like Webflow
- [x]  Resize icons set. Webflow: Icon set is 4 colums. Together 60% width of text block.
- [ ]  Icons width left to right: 50, 51, 46, 47 px.

**Comment by isaachinman** [comment]
I spent a lot of time on this particular section.

It is completely, entirely, utterly broken on the webflow project, so that's not a good starting point.

**Comment by isaachinman** [comment]
c57ed68

**Comment by Tintels** [comment]
TABLET

Screenshot target Webflow site on iPad Mini
![2018-11-23 15 15 23](https://user-images.githubusercontent.com/22452702/48947795-b0e4fb80-ef32-11e8-8781-dc6df73d628b.png)

Screenshot live site on iPad Mini
![2018-11-23 15 15 31](https://user-images.githubusercontent.com/22452702/48947799-b4788280-ef32-11e8-8a01-dac557d90109.png)

Guy and text can be more central / away from the edge

- [ ]  Move guy a bit to the left
- [ ]  Move text a bit to the left

PHONE

Screenshot target Webflow site on Nexus 5X
![screenshot_20181123-151728](https://user-images.githubusercontent.com/22452702/48947870-f7d2f100-ef32-11e8-92a8-883ebbcbf44b.png)

Screenshot live site on Nexus 5X
![screenshot_20181123-151642](https://user-images.githubusercontent.com/22452702/48947875-facde180-ef32-11e8-83a1-ea7cbe27c415.png)

I think you mentioned that you don't like all the rock below the guy. I like it much better / more calm when the guy has similar image area above and below him. Perhaps we can compromise - Crop less rock from the bottom?

Icons size still bit oversized.

- [ ]  No image crop or less crop from the bottom.
- [ ]  Remove drop shadow on title and paragraph
- [ ]  Reduce icon set size. If they will become bit smaller on tablet too, I'll like that.

**Comment by isaachinman** [comment]
A small amount of text-shadow needs to remain for legibility.

**Comment by isaachinman** [comment]
That hiker image is cropped that way, it's not CSS doing the work. I'd need a modified image and I don't see this image in the Dropbox folder.

**Comment by isaachinman** [comment]
32bbd06

**Comment by Tintels** [comment]
There seems to be a bit confusion regarding the image. I actually only used one image for all breakpoints in Webflow and that's why there was only one image in the dropbox folder. I have now created an image for mobile:
https://www.dropbox.com/s/d6qwef2puclm8nv/alif-ngoylung-243373-mobile_2.jpg?dl=0

That one is now also implemented on Webflow so I could check. I cropped some (not all) of the rock below compared to the wide image.
